### PR TITLE
[10.0] better configuration

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

**Comment by hbrunn** [comment]
@NL66278 are you aware the latest commit here fails the tests?

**Comment by NL66278** [comment]
@hbrunn Yes I saw it. Working on solution that will keep the tests correct, but not cause red runbot buildouts due to other tests.

**Comment by NL66278** [comment]
@hbrunn Maybe you want to have another look at this one as well...

File: override_mail_recipients/models/ir_mail_server.py [hunk 1]
```diff
@@ -50,7 +68,7 @@ def patch_message(self, message):
     def send_email(
             self, message, mail_server_id=None, smtp_server=None,
             smtp_port=None, smtp_user=None, smtp_password=None,
-            smtp_encryption=None, smtp_debug=False, smtp_session=None):
+            smtp_encryption=None, smtp_debug=False):
```
Comment (hbrunn) [inline override_mail_recipients/models/ir_mail_server.py:71]: thanks, this slipped in from later versions

**Review by hbrunn — APPROVED** [review]

**Pull request merged**